src/go/rpk/pkg/adminapi/api_debug.go

```diff
@@ -186,3 +193,10 @@ func (a *AdminAPI) DebugPartition(ctx context.Context, namespace, topic string,
 	var response DebugPartition
 	return response, a.sendAny(ctx, http.MethodGet, fmt.Sprintf("/v1/debug/partition/%v/%v/%v", namespace, topic, partitionID), nil, &response)
 }
+
+// RawCPUProfile returns the raw response of the CPU profiler.
+func (a *AdminAPI) RawCPUProfile(ctx context.Context, wait time.Duration) ([]byte, error) {
```

Comment:
+1 for this PR
Maybe if we do Raw a lot, we can just introduce a
```go
func (*AdminAPI) RawRequest(ctx context.Context, pathfmt string, pathargs ...interface{})
```
or some other similar API